src/bin/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# find HTSEngine
+find_path(HTS_ENGINE_INCLUDE_DIR HTS_engine.h)
+find_library(HTS_ENGINE_LIB hts_engine_API)
+if(NOT HTS_ENGINE_LIB)
+  # fallback
+  find_library(HTS_ENGINE_LIB HTSEngine)
+endif()
+
+if(NOT HTS_ENGINE_INCLUDE_DIR OR NOT HTS_ENGINE_LIB)
+  message(FATAL_ERROR "Required HTSEngine not found")
+endif()
+
+add_executable(open_jtalk open_jtalk.c)
+set_target_properties(open_jtalk PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
This property was required in CMake v2.8.12, although it is not required in the CMake v3.16.3.
I haven't checked since when it became unnecessary.